liquigraph-core/src/test/java/org/liquigraph/core/api/LiquigraphTestSuite.java

```diff
@@ -150,4 +153,45 @@ public void call() throws Throwable {
                 .isInstanceOf(IllegalArgumentException.class)
                 .hasMessageContaining("Changeset with ID <second-changelog> and author <team> has conflicted checksums");
     }
+
+    @Test
+    public void runs_migrations_with_parameters() throws SQLException {
+        liquigraph.runMigrations(
+            new ConfigurationBuilder()
+                .withRunMode()
+                .withMasterChangelogLocation("changelog/parameterized-changelog.xml")
+                .withUri(graphDatabase().uri())
+                .build()
+        );
+
+        try (Connection connection = graphDatabase().newConnection()) {
+            try (Statement statement = connection.createStatement();
+                 ResultSet resultSet = statement.executeQuery(
+                 "MATCH (person:Person) RETURN person ORDER BY person.name ASC")) {
+
+                assertThat(resultSet.next()).isTrue();
+                Object person = resultSet.getObject("person");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "person" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)